src/components/Header/index.jsx

```diff
@@ -26,10 +26,16 @@ function Header({ schema, sourceMode, toggleMode }) {
    */
   const classes = useStyles();
 
+  /**
+   * 
+   */
+  const title = schema.title || 'No title';
+  const description = schema.description || 'No description available';
+

```

Comment:
Let's just default to empty strings if they're not defined. I guess you can delete these two lines and just render {schema.title} and {schema.description}.

Reply:
Applied changes!